test3.js

```diff
@@ -0,0 +1,191 @@
+/**
+ * Model file to handle all repository actions
+ */
+const { repoSchema, tempRepoSchema } = require('../schema/index');
+const { Logger, DB } = require('../helpers/index');
+const path = require('path');
+
+/**
+ * Get Repo model from ORM
+ *
+ * @async
+ * @function GetRepoModel
+ * @params {string} - org
+ * @params {boolean} - schedulerEnabled
+ * @returns {object}
+ * @author dev-team
+ */
+
+const GetRepoModel = async (org = null, schedulerEnabled = true) => {
+  try {
+    const model = schedulerEnabled ? { name: 'Repo', schema: repoSchema } : { name: 'TempRepo', schema: tempRepoSchema };
+    Logger.log('info', `Connecting ${model.name} model`);
+    return await DB.GetCollectionModel(org, model.name, model.schema);
+  } catch (exc) {
+    Logger.log('error', `Error in GetRepoModel in ${path.basename(__filename)}: ${JSON.stringify(exc)}`);
+    throw exc;
+  }
+};
+
+/**
+ *  Update Repository based on hooks delivery
+ *
+ * @async
+ * @function UpdateRepository
+ * @params {object}
+ * @returns {object} - array of objects
+ * @author dev-team
+ */
+
+const UpdateRepository = async params => {
+  try {
+    const { RepoModel, query, update, options, deliveryQ, updateHookDelivery } = params;
```

Comment:
Codacy has a fix for the issue: [Insert `⏎·····`](https://app.codacy.com/gh/thamizhMsys/testRepo6/pullRequest?prid=9650715)
```suggestion
    const { RepoModel, query, update, options, deliveryQ, updateHookDelivery } =
```